Example: Add AutomaticKeepAliveClientMixin to FamilyView

rydmike opened the pull request:
As example and demo of case: 
* https://github.com/csells/go_router/issues/45
And as part of documentation for:
* https://github.com/csells/go_router/issues/44

Add the AutomaticKeepAliveClientMixin to the FamilyView example to demonstrate how to keep state.

csells commented:
love this update, @rydmike! can you also add a section to the README re: keeping state on pages during navigation?

rydmike commented:
Sure, will do tomorrow, 2:30 am here now, gotta catch some ZzZ now. :)

rydmike commented:
@csell I added a section to the readme about "Keeping State" after the "Nested Navigaation" section.
Also fixed some typos my IDE spellchecker suggested in the readme.

Feel free to review or edit.

I also added an animated GIF showing it working and keeping state with the nest tab view nav. However , due to the few number of people in the families it is not so clear. If we add a lot more people to the families it would be easier to see and demonstrate that it is working, but I did not want to mess with your family data! :D
